src/Discord.Net.Core/Entities/IApplication.cs

```diff
@@ -21,7 +21,9 @@ public interface IApplication : ISnowflakeEntity
         /// <summary>
         ///     Gets the icon URL of the application.
         /// </summary>
-        string IconUrl  { get; }
+        /// <param name="format">The format to return. Mustn't be a gif.</param>
```

Comment:
```suggestion
        /// <param name="format">The format to return. Must not be of type <see cref="ImageFormat.Gif"/></param>
```
Refers to the type, which should have a link in docs.